test/controllers/declaration/AdditionalInformationControllerSpec.scala

```diff
@@ -91,6 +103,8 @@ class AdditionalInformationControllerSpec
         val result = route(app, postRequestFormUrlEncoded(uri, body: _*)).get
 
         status(result) must be(SEE_OTHER)
+
+        verify(mockExportsCacheService, times(1)).update(any(), any())
```

Comment:
I'm pretty sure you  don't need the `times(1)` as *I think* that is the default for verify.

Also do  you think it's worth doing a more exhaustive verify given theres some mapping code that currently isn't tested?